docs/Benchmarks.md

```diff
@@ -0,0 +1,94 @@
+# Benchmarks
+
+This document explains how to benchmark ASP.NET Core changes. It is intended for Microsoft employees.
+It describes how to trigger benchmarks using the Crank CLI and the Pull-Request bot.
+
+## Benchmarking local changes
+
+This is the recommended method as it is quick, flexible and doesn't require a pre-existing pull-request.
+
+### Install Crank
+
+The Crank dotnet tool is required to start benchmarks remotely on the available performance infrastructure.

```

Comment:
Maybe briefly explaining what Crank is -- what it can do, what it supports (eg., architectures), how long jobs typically take, ..

Personally I'm also curious where the name came from...